core/block_svg.js

```diff
@@ -655,6 +655,87 @@ Blockly.BlockSvg.prototype.showHelp_ = function() {
 };
 
 /**
+ * Creates a callback function for a click on the "duplicate" context menu
+ * option in Scratch Blocks.  The block is duplicated and attached to the mouse,
+ * which acts as though it were pressed and mid-drag.  Clicking the mouse
+ * releases the new dragging block.
+ * @return {Function} A callback function that duplicates the block and starts a
+ *     drag.
+ * @private
+ */
+Blockly.BlockSvg.prototype.duplicateAndDragCallback_ = function() {
+  var oldBlock = this;
+  return function(e) {
+    // Give the context menu a chance to close.
+    setTimeout(function() {
+      var ws = oldBlock.workspace;
+      var svgRootOld = oldBlock.getSvgRoot();
+      if (!svgRootOld) {
+        throw 'oldBlock is not rendered.';
```

Comment:
Do you know which eslint rule offhand?  If not I'll look into it.

I'm fine with using Errors.